.drone.yml

```diff
@@ -42,6 +42,35 @@ steps:
     event:
       - push
 
+- name: docker-build-sle-micro
```

Comment:
Could we trigger PR build according to branches?
`dev` -> `docker-build`
`sle-micro` -> `docker-build-sle-micro`

Reply:
I think I forgot some "when" conditions.
let me deal with it.

Reply:
let me check it.

Reply:
It should work now.
This PR is built by `docker-build-sle-micro` task